projects/plugins/crm/package.json

```
@@ -1,29 +1,34 @@
 {
 	"name": "@automattic/jetpack-crm",
-	"version": "5.5.2-alpha",
+	"version": "5.6.0-alpha",
 	"description": "The CRM for WordPress",
 	"author": "Automattic",
 	"license": "GPL-2.0",
 	"scripts": {
-		"start": "npx wp-env start",
-		"stop": "npx wp-env stop",
-		"clean": "npx wp-env clean all"
+		"build": "pnpm run clean && pnpm run build-client",
+		"build-client": "NODE_PATH=\"$PWD/node_modules\" webpack",
+		"build-production": "NODE_ENV=production BABEL_ENV=production pnpm run build",
+		"clean": "rm -rf **/css/**/*.rtl.* .cache/ **/css/*.min.css **/js/*.min.js.* **/js/*.min.js  **/js/**/*.min.js.map **/css/**/*.css.map",
```

The `**` here probably doesn't work right. Bash by default doesn't have globstar enabled.

This seems to work
```suggestion
		"clean": "rm -rf css/*.min.css* css/*.min.rtl.css css/integrations/*.min.css* css/integrations/*.min.rtl.css css/welcome-to-zbs/*.min.css* css/welcome-to-zbs/*.min.rtl.css js/*.min.js* js/welcome-to-zbs/wizard2.min.js*",
```
It looks like there are still some `.min.css` files checked in or we could do like
```suggestion
		"clean": "find css js modules \( -name '*.min.css*' -o -name '*.min.rtl.css' -o -name '*.min.js*' \) -delete"
```

---

> It looks like there are still some .min.css files checked in or we could do like

Yes that would be a good 'next' project, to do the same for all folders as was done with welcome-to-zbs, depending on what the CRM preferences are.

> The ** here probably doesn't work right. Bash by default doesn't have globstar enabled.

Ah! Must have been a setting at my end that helped it work. Thank you. I'll commit a new change, as it looks even better to just delete the whole /integrations folder (only created when the CSS is being compiled and minified now).